test/api_test.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:movies_project/src/model/movies.dart';
+import 'package:movies_project/src/resources/movies_api_provider.dart';
+
+void main() {
+  late MoviesApiProvider moviesApiProvider;
+  setUp(
+    () {
+      moviesApiProvider = MoviesApiProvider();
+    },
+  );
+
+  group(
+    "Api testing",
+    () {
+      test(
+        "Trending Movies",
```

Comment:
Try to add a more descriptive test name like "Should be fetch trending movies when fetchTrendingMovies is call" or something like that

Reply:
done